bin/nanocld/src/utils/cargo.rs

```diff
@@ -887,20 +887,12 @@ pub async fn patch(
 
 pub fn get_logs(
   name: &str,
-  query: &CargoLogQuery,
+  query: CargoLogQuery,
```

Comment:
You should keep the reference there `&CargoLogQuery` and do a `query.clone().into()`

Reply:
whats the reason for doing so? Would there not always be 2 instances of the object. While when its up for the parent function to decide if a clone should happen there could be only 1 instance